tests/components/tessie/test_binary_sensors.py

```diff
@@ -16,18 +15,12 @@ async def test_binary_sensors(hass: HomeAssistant) -> None:
 
     await setup_platform(hass)
 
-    assert len(hass.states.async_all("binary_sensor")) == len(DESCRIPTIONS)
```

Comment:
Please use `@pytest.mark.usefixtures("entity_registry_enabled_by_default") so you can also test disabled entities.